tests/testthat/test-emissions_profile.R

```diff
@@ -385,6 +385,50 @@ test_that("at company level, two matched and one unmatched products yield `value
   expect_equal(sort(other), c(0, 0, 2 / 3))
 })
 
+test_that("At company level, three products with different `co2_footprint` yields `value = 1/3` where `risk_category` is 'high', 'medium', and 'low'", {
+  companies <- example_companies(
+    !!aka("uid") := letters[1:3],
+  )
+
+  co2 <- example_products(
+    !!aka("uid") := letters[1:3],
+    !!aka("co2footprint") := 1:3,
+  )
+
+  out <- emissions_profile(companies, co2) |> unnest_company()
+
+  actual <- out |>
+    filter(value != 0) |>
+    distinct(risk_category, value)
+
+  expect_equal(actual$risk_category, c("high", "medium", "low"))
+  expect_true(all(actual$value != 0))
+  expect_equal(unique(actual$value), 1 / 3)
+})
```

Comment:
@Tilmon this is the test-version of the reprex we discussed [here](https://github.com/2DegreesInvesting/tiltIndicator/issues/730#issuecomment-1961527398). It's here because it adds an important expectation into our code base, and also to help you understand how a conceptual reprex translates to a technical test.